README.md

```diff
@@ -11,7 +11,7 @@ status](https://www.r-pkg.org/badges/version/crmPack)](https://CRAN.R-project.or
 [![Project Status: Active - The project has reached a stable, usable
 state and is being actively
 developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
-[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/license/apache-2-0)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
needs to be regenerated to be consistent with the `Rmd` file regarding this line?